### PR TITLE
Update BlazorWasm3.2P1Updates

### LINKED ISSUE #16761: Blazor WebAssembly 3.2 Preview 1 Updates

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #16761

* WebAssembly sample app updated and confirmed working.
* Can you explain the 'no overload of 'GetRequiredService' takes 1 argument' error for `var weatherService = host.Services.GetRequiredService<WeatherService>(host.Configuration["WeatherServiceUrl"]);` from the blog post? Is there a more complete example somewhere? I didn't see one in the engineering repo (e.g., in the tests).
* The *Hosting Models* topic describes the hosting models but also goes into advanced configuration details that might be better off in another topic. Today, this is the only good spot for that configuration/advanced content. I've opened https://github.com/aspnet/AspNetCore.Docs/issues/16764 to discuss/triage.

### CODE REVIEW

File: aspnetcore/blazor/hosting-models.md [hunk 1]
```diff
@@ -48,6 +48,121 @@ There are downsides to Blazor WebAssembly hosting:
 * Download size is larger, and apps take longer to load.
 * .NET runtime and tooling support is less mature. For example, limitations exist in [.NET Standard](/dotnet/standard/net-standard) support and debugging.
 
+Blazor WebAssembly supports the [.NET SignalR client](xref:signalr/dotnet-client). To use SignalR in a Blazor WebAssembly app:
```
Comment (danroth27) [inline aspnetcore/blazor/hosting-models.md]: I would prefer for this content to get integrated into the getting started tutorial for SignalR. Currently, that tutorial shows how to use the Signalr JS client. We could add tabs to that tutorial that show you how to use the .NET client from the browser instead if you are running a Blazor WebAssembly app. @bradygaster What do you think?

Comment (danroth27) [reply aspnetcore/blazor/hosting-models.md]: Actually, it looks like we already have separate tutorials for JS & TS, so we could just add another one for .NET with Blazor.

Comment (danroth27) [reply aspnetcore/blazor/hosting-models.md]: https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr

Comment (guardrex) [reply aspnetcore/blazor/hosting-models.md]: I'll get this resolved.

Comment (danroth27) [reply aspnetcore/blazor/hosting-models.md]: @guardrex Until we get agreement with the SignalR team I think we should pull this content out of this PR and treat it separately.

Comment (guardrex) [reply aspnetcore/blazor/hosting-models.md]: Ok ... I'll open an issue then on hold.

Comment (guardrex) [reply aspnetcore/blazor/hosting-models.md]: I'm going to hold it on this current issue that I opened for what to do with config content until we have a home for it ...

https://github.com/aspnet/AspNetCore.Docs/issues/16764#issuecomment-579916238

**Comment by danroth27** [comment]
> Can you explain the 'no overload of 'GetRequiredService' takes 1 argument' error for var weatherService = host.Services.GetRequiredService<WeatherService>(host.Configuration["WeatherServiceUrl"]); from the blog post? Is there a more complete example somewhere? I didn't see one in the engineering repo (e.g., in the tests).

This was a typo in the blog post. The config value should have been passed to the hypothetical `InitializeWeatherService` method. Should be fixed now.

**Pull request merged**